mergify_engine/redis_utils.py

```diff
@@ -67,12 +67,13 @@ def register_script(script: str) -> ScriptIdT:
 # it works but if a script is loaded into two redis, this won't works as expected
 # as the app will think it's already loaded while it's not...
 async def load_script(
-    redis: typing.Union[RedisCache, RedisStream], script_id: ScriptIdT
+    redis: "redispy.connection.Connection", script_id: ScriptIdT
 ) -> None:
     global SCRIPTS
     sha, script = SCRIPTS[script_id]
-    # FIXME(sileht): weird, this method is typed on redis-py
-    newsha = await redis.script_load(script)  # type: ignore[no-untyped-call]
+    await redis.send_command("SCRIPT LOAD", script)
+    response = await redis.read_response()
+    newsha = response.decode("utf-8")
```

Comment:
1. decoding as ASCII might be enough
2. you could rather encode the sha1 computed in L60; that'd make encoding once and compare N times, rather than encoding+compare N times

Reply:
Done